partiql-ast/src/main/java/org/partiql/ast/Let.java

```diff
@@ -47,9 +47,9 @@ public static class Binding extends AstNode {
         private final Expr expr;
 
         @NotNull
-        private final Identifier asAlias;
+        private final Identifier.Part asAlias;
```

Comment:
Hmm seeing all of this makes me thing "Identifier.Single" or "Identifier.Simple" is better than Part. I leave it entirely up to your opinion and John's – my only requirement is that any change is also made to SPI so that these are the same. As I said earlier, I don't love "part" and seeing the single/simple identifier used in many more places has inspired a better definition

Reply:
Naming is definitely hard 😅. I'll try to get some consensus here before adjusting in the AST + SPI

Reply:
Just my opinion though they are all reasonable options:
- part
  - nothing wrong w/ it,
  - makes it clear that it's just a segment of an Identifier
  - an `Identifier` is built up from a list of parts (a qualifier that's a list of parts and a base part)
- single
  - could be confused for an individual identifier which could be qualified
- simple
  - I think slightly better than `single`
  - better for the base identifier but may be more confusing for the qualifier (i.e. a qualifier is a list of simple identifiers)

Reply:
SQL's `<identifier>` is unqualified. And `<table name>`, `<local or schema qualified name>`, and `<identifier chain>` are all (potentially) qualified identifiers.

If we want `Identifier` to be represent the qualified identifier, then I think I like `Identifer.Simple` with some good Javadocs telling the difference between the two. An alternative is two have two classes: `IdentQualified/IdentifierQualified` and `Ident/Identifier`. Though, I'm fine with whatever approach you take @alancai98.

Reply:
Changed to `Identifier.Simple` in the AST and SPI. Perhaps it's overkill to create two separate classes (like the v0.14.* modeling), so kept it as a nested class within `Identifier`.